tutorials/tts/FastPitch_ChineseTTS_Training.ipynb

```diff
@@ -541,45 +449,45 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "4ad43d30",
+   "id": "af763ead",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "!cd NeMoChineseTTS && CUDA_VISIBLE_DEVICES=0 python3 fastpitch.py --config-path . --config-name fastpitch_align_22050 \\\n",
+    "!cd NeMoChineseTTS && CUDA_VISIBLE_DEVICES=0 python fastpitch.py --config-path . --config-name fastpitch_align_22050 \\\n",
     "    model.train_ds.dataloader_params.batch_size=32 \\\n",
-    "    model.validation_ds.dataloader_params.batch_size=8 \\\n",
-    "    train_dataset=./train_manifest.json \\\n",
-    "    validation_datasets=./val_manifest.json \\\n",
-    "    sup_data_path=./sup-data-phonemes \\\n",
-    "    whitelist_path=./default.tsv \\\n",
+    "    model.validation_ds.dataloader_params.batch_size=32 \\\n",
+    "    train_dataset=train_manifest.json \\\n",
+    "    validation_datasets=val_manifest.json \\\n",
+    "    sup_data_path=sup_data \\\n",
+    "    whitelist_path=default.tsv \\\n",
     "    exp_manager.exp_dir=resultChineseTTS \\\n",
     "    trainer.max_epochs=1 \\\n",
     "    trainer.check_val_every_n_epoch=1 \\\n",
-    "    pitch_mean=221.4948272705078 \\  # Paste the PITCH_MEAN here.\n",
-    "    pitch_std=64.65289306640625 \\  # Paste the PITCH_STD here.\n",
-    "    phoneme_dict_path=./pinyin_dict_nv_22.10.txt \\\n",
+    "    pitch_mean=226.75924682617188 \\\n",
+    "    pitch_std=58.773109436035156 \\\n",
```

Comment:
Why get rid of the comment from before?

Reply:
we want users to override these variables based on what they obtained in the previous step. I added these values as a reference in order to make our tutorial run automatically. I would add more guides.

Reply:
Keeping the old comments (like `# Paste the PITCH_MEAN here.`) might still be useful for users, don't you think?

Reply:
yes, but it would lead to an error if appending with `# xyz` in the command line in the cell. That's why I removed them.

Reply:
Ah, gotcha. Sounds good then.